Without this it tries to include the system drm/drm.h which won't work.
--- a/lact-daemon/build.rs
+++ b/lact-daemon/build.rs
@@ -20,6 +20,7 @@
 
     bindgen::builder()
         .header("include/intel.h")
+        .clang_arg("-Iinclude")
         .parse_callbacks(Box::new(bindgen::CargoCallbacks::new()))
         .dynamic_library_name("IntelDrm")
         // Don't generate bindings for functions and types related to file access
@@ -39,6 +40,7 @@
 
     bindgen::builder()
         .header("include/nvidia.h")
+        .clang_arg("-Iinclude")
         .parse_callbacks(Box::new(bindgen::CargoCallbacks::new()))
         .generate_comments(false)
         .clang_arg("-Iinclude/nvidia/src/common/sdk/nvidia/inc")
